vue/sbc-common-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sbc-common-components",
-  "version": "2.3.1",
+  "version": "2.3.3",
```

Comment:
I'm expecting Nitheesh's PR to change this to 2.3.2 and be merged in before THIS PR.

Reply:
its merged now